src/Compilers/VisualBasic/Portable/Symbols/ParameterSymbol.vb

```diff
@@ -266,6 +266,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Friend MustOverride ReadOnly Property IsCallerFilePath As Boolean
 
+        Friend MustOverride ReadOnly Property CallerArgumentExpressionParameterIndex As Integer
```

Comment:
>Friend MustOverride ReadOnly Property CallerArgumentExpressionParameterIndex As Integer [](http://example.com/codeflow?start=8&length=87)

Given the special range of values for extension methods, I think it is worth documenting what values to expect and how to interpret them